build-logic/kotlin-dsl-shared-runtime/src/main/kotlin/org/gradle/kotlin/dsl/internal/sharedruntime/codegen/SourceFileHeader.kt

```diff
@@ -33,22 +33,23 @@ fun fileHeaderFor(packageName: String, isIncubating: Boolean = false) =
     "redundant_projection",
     "RemoveRedundantBackticks",
     "ObjectPropertyName",
-    "deprecation"
+    "deprecation",
+    "detekt:all"
 )
 @file:org.gradle.api.Generated${if (isIncubating) "\n@file:org.gradle.api.Incubating" else ""}
 
-/* ktlint-disable */
-
 package $packageName
 """
 
-
+@Suppress("TopLevelPropertyNaming") // TODO: renaming leads to "Unresolved reference" error at use sites... why is that?
```

Comment:
Yes, but figuring it out failed so far. It's tricky. Haven't given up completely yet.

Reply:
I'm unable to figure this out... @eskatos or @bamboo , can you please advise?

Reply:
This is because this property resides in a module that is used both by build logic and runtime.

In order to change its name you'd need to:
- add a new property with the new name
- update the wrapper
- remove the old property

Reply:
Ok, I've added the new constants, once this PR is merged in I will do the wrapper replacement and old constant replacement in a separate PR.